<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-N18APC6D" failures="5" tests="19" name="Test" time="10.425" errors="0" timestamp="2022-07-25T17:16:34 CDT">
  <testcase classname="createOrModifyKeyValuePairAPI.GetAllKeys" name="GetAllKey" time="1.628"/>
  <testcase classname="createOrModifyKeyValuePairAPI.GetAllKeys" name="GetAllKeyWithOutAuth" time="0.637"/>
  <testcase classname="createOrModifyKeyValuePairAPI.GetAllKeys" name="GetAllKeyInvalidID" time="0.986"/>
  <testcase classname="createOrModifyKeyValuePairAPI.DeleteAKeyValuePair" name="DeleteKeyValue" time="0.619"/>
  <testcase classname="createOrModifyKeyValuePairAPI.DeleteAKeyValuePair" name="ValidateDeletedKeyUsingGet" time="0.946">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at createOrModifyKeyValuePairAPI.DeleteAKeyValuePair.ValidateDeletedKeyUsingGet(DeleteAKeyValuePair.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ValidateDeletedKeyUsingGet -->
  <testcase classname="createOrModifyKeyValuePairAPI.DeleteAKeyValuePair" name="DeleteKeyValueWithOutAuth" time="0.829"/>
  <testcase classname="createOrModifyKeyValuePairAPI.DeleteAKeyValuePair" name="DeleteKeyValueWithInvalidID" time="0.548">
    <failure type="java.lang.AssertionError" message="expected [404] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at createOrModifyKeyValuePairAPI.DeleteAKeyValuePair.DeleteKeyValueWithInvalidID(DeleteAKeyValuePair.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- DeleteKeyValueWithInvalidID -->
  <testcase classname="createOrModifyKeyValuePairAPI.GetAValue" name="GetPerticularKeyValue" time="0.836"/>
  <testcase classname="createOrModifyKeyValuePairAPI.GetAValue" name="GetPerticularKeyValueWithOutAuth" time="0.165"/>
  <testcase classname="createOrModifyKeyValuePairAPI.GetAValue" name="GetPerticularKeyValueWithInvalidData" time="0.734"/>
  <testcase classname="createDictionaryAPI.CreateAnewDictionary" name="CreateDictionary" time="0.262"/>
  <testcase classname="createDictionaryAPI.CreateAnewDictionary" name="CreateDictionarywithNoAuthorization" time="0.183"/>
  <testcase classname="createDictionaryAPI.CreateOrModifyAkeyValuePair" name="CreateAkeyValuePair" time="0.193"/>
  <testcase classname="createDictionaryAPI.CreateOrModifyAkeyValuePair" name="CreateAkeyValuePairWithOutAuthorization" time="0.155"/>
  <testcase classname="createDictionaryAPI.CreateOrModifyAkeyValuePair" name="ModifyAkeyValuePair" time="0.151">
    <failure type="java.lang.AssertionError" message="expected [201] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at createDictionaryAPI.CreateOrModifyAkeyValuePair.ModifyAkeyValuePair(CreateOrModifyAkeyValuePair.java:97)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ModifyAkeyValuePair -->
  <testcase classname="deleteDictinaryAPI.DeleteAdictionary" name="DeleteDictionary" time="0.253">
    <failure type="java.lang.AssertionError" message="expected [200] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [204]
at deleteDictinaryAPI.DeleteAdictionary.DeleteDictionary(DeleteAdictionary.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- DeleteDictionary -->
  <testcase classname="deleteDictinaryAPI.DeleteAdictionary" name="ValidateDeletedKeyUsingGet" time="0.228">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at deleteDictinaryAPI.DeleteAdictionary.ValidateDeletedKeyUsingGet(DeleteAdictionary.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ValidateDeletedKeyUsingGet -->
  <testcase classname="deleteDictinaryAPI.DeleteAdictionary" name="DeleteDictionaryWithOutAuthorization" time="0.218"/>
  <testcase classname="deleteDictinaryAPI.DeleteAdictionary" name="DeleteDictionaryWithInvalidID" time="0.285"/>
</testsuite> <!-- Test -->
